bot.py

```diff
@@ -176,14 +185,14 @@ async def clear_prefix(self, guild: discord.Guild) -> None:
         ------
         PrefixNotInitialised
             Raised when prefixes for the guild were never created.
+
         """
```

Comment:
did you make an indent mistake here?

Reply:
Seems to be done by ruff formatter since there are similar indents in this file. Does it not comply with docstring rules that I'm not aware of? Assuming there are any set by this doc style